framework/actions/aggregate_test.go

```diff
@@ -89,6 +94,79 @@ func TestAggregateResults(t *testing.T) {
 	require.Len(t, gotResults, 1)
 }
 
+func TestAggregateResults_Multi(t *testing.T) {
```

Comment:
A test scenario where one plugin fails could be good to see this behavior in our tests.